src/NuGet.Clients/NuGet.CommandLine/CommandLineMachineWideSettings.cs

```diff
@@ -8,17 +8,23 @@
 namespace NuGet.CommandLine
 {
     [Export(typeof(Configuration.IMachineWideSettings))]
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
@zivkan @nkolev92 continuing the conversation here since GitHub wouldn't let me reopen the PR. 

Since this project is shipped on nuget.org, we should keep the warning right?

Reply:
The NuGet.CommandLine package is a transport vehicle for nuget.exe, which is used by `nuget.exe update -self`.

It's not a library that customers can reference in their project and call NuGet APIs on. Therefore, XML comments only benefit people working within this repo, but will no benefit anyone else.

Reply:
So this is not the same as this package? https://www.nuget.org/packages/NuGet.CommandLine

Reply:
it is the same package. My previous comment still applies 😄

Reply:
Should we disable the xml comments for NuGet.Commandline since it's not really shipping API?

Reply:
I added a new .editorconfig to the project and set this warning as a suggestion. When I added the .editorconfig file it made a couple of changes to the csproj so I removed them... 

Not sure why it was adding a <EditorConfigFiles Remove=".editorconfig"/> by default O.o

Do I need the <None Include=".editorconfig" /> though? I removed it and everything seems fine 🤷

Reply:
Personally, idk. I didn't know editor configs could nest. 

Is there a way to change that through msbuild potentially?

Reply:
oo duh, added a nowarn, much less confusion :)